lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -1,27 +1,35 @@
 defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
   @moduledoc false
 
+  alias Screens.Alerts.Alert
   alias Screens.Config.Screen
   alias Screens.Config.V2.Header.CurrentStopId
   alias Screens.Config.V2.PreFare
+  alias Screens.Routes.Route
   alias Screens.V2.WidgetInstance.ReconstructedAlert
 
-  def reconstructed_alert_instances(%Screen{
-        app_params: %PreFare{header: %CurrentStopId{stop_id: _stop_id}}
-      }) do
-    # Given the stop id, determine relevant routes
-    temp_route_ids = ["Red", "Green-B", "Green-C", "Green-D", "Green-E"]
+  @relevant_effects ~w[shuttle suspension station_closure delay]a
 
-    # Given route ids, fetch alerts
-    case Screens.Alerts.Alert.fetch(route_ids: temp_route_ids) do
-      {:ok, alerts} ->
-        Enum.map(
-          alerts,
-          fn alert -> %ReconstructedAlert{alert: alert} end
-        )
-
-      :error ->
-        []
+  @doc """
+  Given the stop_id defined in the header, determine relevant routes
+  Given the routes, fetch all alerts for the route
+  """
+  def reconstructed_alert_instances(
+        %Screen{app_params: %PreFare{header: %CurrentStopId{stop_id: stop_id}}},
```

Comment:
Eventually we'll want to use a separate config definition for the reconstructed alert widget, but this makes sense for now!

Reply:
Ah, why? I don't foresee other values we'd need,  so a concise config seems useful.

Reply:
(Discussed over a slack huddle—makes sense to avoid duplicating config values just because they're used by different widgets! We will probably try to define a common "screen location context" structure that all widgets can use as needed)